scripts/convert_original_stable_diffusion_to_diffusers.py

```diff
@@ -65,7 +68,7 @@
         type=str,
         help=(
             "The prediction type that the model was trained on. Use 'epsilon' for Stable Diffusion v1.X and Stable"
-            " Siffusion v2 Base. Use 'v-prediction' for Stable Diffusion v2."
+            " Diffusion v2 Base. Use 'v_prediction' for Stable Diffusion v2."
```

Comment:
https://github.com/huggingface/diffusers/blob/a7ca03aa85f94574f06576d2155b3ec061fe8d63/src/diffusers/schedulers/scheduling_ddim.py#L278

The scheduler takes 'v_prediction'

Reply:
Good catch!